src/components/Banner/index.jsx

```diff
@@ -0,0 +1,45 @@
+import styles from "./banner.module.css";
+import logoImage from "../../images/logo.png";
+import vectorImage from "../../images/vector.png";
+import Typography from "@mui/material/Typography";
+import { SearchBar } from "./SearchBar/index.jsx";
+import { LandingPageLink } from "./LandingPageLink/index.jsx";
+import { SearchButton } from "./SearchButton/index.jsx";
+
+export function Banner({ size }) {
+  let bannerHeight = 0;
+  if (size === "s") {
+    bannerHeight = "20.83vw";
+  } else if (size === "m") {
+    bannerHeight = "36.38vw";
+  } else if (size === "l") {
+    bannerHeight = "52.15vw";
+  }
+
+  return (
+    <div className={styles.container}>
+      <div
+        className={styles.wrapper}
+        style={{ "--banner-height": bannerHeight }}
```

Comment:
Inline styles along with !important and position absolute is one of the worst things to do in the code 😰

Reply:
done